src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -59,21 +59,23 @@ interface ConfirmBidProps extends ReactStripeElements.InjectedStripeProps {
 const MAX_POLL_ATTEMPTS = 20
 
 export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
+  let timer = null
   let pollCount = 0
   let registrationTracked = false
 
   const { artwork, me, relay, stripe } = props
   const { saleArtwork } = artwork
   const { sale } = saleArtwork
+  const { registrationStatus } = sale as any
```

Comment:
similar question to below. I know that in the red diff `sale` was cast to `any` – but pulling down the code in VScode indicates that `saleArtwork` is typed. Have we tried to remove the `any`?